app/renderer/js/tray.ts

```diff
@@ -183,6 +185,34 @@ ipcRenderer.on('tray', (_event: Event, arg: number): void => {
 	}
 });
 
+const monitorTray = (): void => {
+	if (process.platform !== 'linux') {
+		return;
+	}
+
+	/* Here we check if the screen has been locked and unlocked.
+		If the monitor returns SCREEN_UNLOCKED, then that has to be our case
+		And we destroy and recreate the tray
+	*/
```

Comment:
Can you please add comments here linking to the upstream issue and mentioning that we can delete this workaround once upstream resolves the issue?  That will help maintainers a lot in the future.

Reply:
Sure thing. Will be adding in the next commit